install/create-docker-volumes-test.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+source "$(dirname $0)/_test_setup.sh"
+
+expected=7
+count() {
+  docker volume ls --quiet | grep '^sentry-.*' | wc -l
```

Comment:
This actually highlights the need to get this list out somewhere. I think we can do the following:

- Rename all internal volumes in `docker-compose.yml` to *NOT* have the `sentry-` prefix
- Run `docker-compose config --volumes` and filter it with `grep '^sentry-'`
- Use this list to create all global volumes
- Use this list to verify all volumes created

What do you think?

Reply:
A similar thought occurred to me but I want to avoid scope creep. If/when we land #795 entirely that will give us a solid foundation for doing more refactors like this one in the future.

Reply:
Just realized renaming these volumes may cause issues during upgrades so we may need to find an alternative method anyway.

Reply:
Right, didn't want to go down that 🐰 🕳️ here! 🙈